server/internal/client/ollama/registry.go

```diff
@@ -595,6 +610,9 @@ type Manifest struct {
 	Name   string   `json:"-"` // the canonical name of the model
 	Data   []byte   `json:"-"` // the raw data of the manifest
 	Layers []*Layer `json:"layers"`
+
+	// For legacy reasons, we still have to download the config layer.
```

Comment:
Whats the reason?

Reply:
I didn't look to hard, but there is plenty of code touching and looking for a config, all of which break if I don't add a config, so I added the config, because ollama likes configs and wants a config of its config.